api/app/src/external/commonwell/patient.ts

```diff
@@ -73,14 +76,17 @@ export async function create(patient: Patient, facilityId: string): Promise<void
     storeIds,
   });
 
-  await findOrCreatePersonAndLink({
-    commonWell,
-    queryMeta,
-    commonwellPatient,
-    commonwellPatientId,
-    patientRefLink,
-    storeIds,
-  });
+  // Note; Elaborate in PR
+  if (!Config.isSandbox()) {
+    await findOrCreatePersonAndLink({
+      commonWell,
+      queryMeta,
+      commonwellPatient,
+      commonwellPatientId,
+      patientRefLink,
+      storeIds,
+    });
+  }
```

Comment:
So this function calls many other functions that contain commonwell actions. I was stubbing each one of those responses but then realized that this function is really only a wrapper for commonwell and doesnt interact with our system except for the line below which i think is fine but let me know what you guys think.

`await storeIds({ commonwellPatientId, personId });`

Reply:
Yeah that makes sense - note my other comment regarding the design of this

Reply:
I think it would be good to execute that logic too, which aligns with the approach we're taking forward of mocking the API - so the main logic of the app will be the same, just the API behavior changing.